src/appleseed.shaders/src/maya/as_maya_anisotropic.osl

```diff
@@ -269,7 +269,7 @@ shader as_maya_anisotropic
                 );
         }
 
-        if (transparency)
+        if (max(transparency) > 0.0)
```

Comment:
All this is constant folded by OSL in most cases. There's usually no runtime cost.